examples/main.go

```diff
@@ -0,0 +1,56 @@
+package main
+
+import (
+	"os"
+	"reflect"
+
+	"github.com/digitalocean/firebolt/util"
+
```

Comment:
not that this matters at all, but should we run `goimports` or w/e the new thing is in CI to make the imports conform to the standard

Reply:
fixed in next commit!